user/views.py

```diff
@@ -1,3 +1,126 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import re
+import hashlib
+import hmac
+import base64
+import requests
+import time
+import random
+import jwt
+import datetime
+
+from json.decoder     import JSONDecodeError
+
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models          import User
+from my_settings      import SMS_SERVICE_ID, SMS_SECRET_KEY, SMS_ACCESS_KEY, HASHING_ALGORITHM, SECRET_KEY
+from .validator        import Validator
+
+class SignUpView(View):
+    @Validator
+    def post(self, request):
+        try:
+            data          = json.loads(request.body)
+            email         = data['email']
+            password      = data['password']
+
+            email_check = User.objects.filter(email=email).exists()
+            email_regex = re.compile(r'^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9_-]+\.[a-zA-Z-.]+$')
+          
+            password_regex = re.compile(r'^(?=.*[!-/:-@])(?!.*[ㄱ-ㅣ가-힣]).{8,20}$')
+      
+
+            if email_check:
+                return JsonResponse({'message': 'error_email_exists'}, status=401)
```

Comment:
```suggestion
                return JsonResponse({'message': 'ERROR_EMAIL_EXISTS'}, status=401)
```
우리가 status 코드로 이미 설명이 되는 에러에  메세지를 주는 이유는 같은  status 코드에서도 분별력있는 판단이 되도록 하는 것 입니다. 다만 이를 대문자 및 단어 구분시 '_'로 이어주는 컨벤션으로 에러 메세지도 하나의 코드로서 관리하려고 안내 드린겁니다~! 그러므로 에러 메세지는 간결하고 메세지 코드로서 강조드린 컨벤션대로 작성 부탁드립니다!

Reply:
수정하겠습니다.